cheatsheets/Password_Storage_Cheat_Sheet.md

```diff
@@ -142,6 +142,12 @@ The work factor for PBKDF2 is implemented through an iteration count, which shou
 - PBKDF2-HMAC-SHA256: 600,000 iterations
 - PBKDF2-HMAC-SHA512: 210,000 iterations
 
+### Parallel PBKDF2
+
+- PPBKDF2-SHA512: cost 2
+- PPBKDF2-SHA256: cost 5
+- PPBKDF2-SHA1: cost 10
+
```

Comment:
Can someone more well versed in this topic please review this?

Reply:
https://infosec.exchange/@sc00bz/109599362314030488 and https://tobtu.com/minimum-password-settings/
You can go through these in order to get a reference on what's going on as stated in issue #1043